index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <h2>John Omachi</h2>
+        <img src="./images/hamburger.png" alt="image" />
+      </nav>
+    </header>
+    <section class="jumbotron">
+      <h1>John Omachi</h1>
+      <h2>Software Developer</h2>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+    </section>
+    <div class="prog-lang-skills">
+      <div class="prog-lan">
+        <ul>
+          <li class="list-head">Language</li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul>
+          <li class="list-head">Frameworks</li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rail</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul>
+          <li class="list-head">Skills</li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="social-media">
+      <div class="hidden">
+        <p>work</p>
+      </div>
```

Comment:
- [OPTIONAL] I see that you don't have to add this div here, I suggest you to delete this part. (You already added it down 😅)

Reply:
Thank you so much for the review.

I added that div to the start of the section and the end, So that I can display flex and justify-content: space-beetween; and also hide the first and last one in other to keep the ul in the middle so as the resolve the errors of my images floatingto the right side of my page when screen resolution is reduced, to solve the error pointed out earlier about re-aligning my social media icons proper.